{"uuid": "e2a529ea-468d-4e2e-8e41-f2057b47787e", "children": ["fd6a1017-ded7-4d63-8c64-9f881bfadaa9"], "befores": [{"name": "authenticated_user", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: target frame detached: received Inspector.detached event\n  (failed to check if window was closed: disconnected: Unable to receive message from renderer)\n  (Session info: chrome=132.0.6834.196)\nStacktrace:\n\tGetHandleVerifier [0x00E774A3+25091]\n\t(No symbol) [0x00DFDC04]\n\t(No symbol) [0x00CDB373]\n\t(No symbol) [0x00CCDEE8]\n\t(No symbol) [0x00CCD107]\n\t(No symbol) [0x00CCCC6B]\n\t(No symbol) [0x00CCC981]\n\t(No symbol) [0x00CCBAE9]\n\t(No symbol) [0x00CE6901]\n\t(No symbol) [0x00D5B826]\n\t(No symbol) [0x00D41CA6]\n\t(No symbol) [0x00D131D5]\n\t(No symbol) [0x00D1435D]\n\tGetHandleVerifier [0x011707C3+3142947]\n\tGetHandleVerifier [0x01181A2B+3213195]\n\tGetHandleVerifier [0x0117C412+3191154]\n\tGetHandleVerifier [0x00F18720+685184]\n\t(No symbol) [0x00E06E1D]\n\t(No symbol) [0x00E03E18]\n\t(No symbol) [0x00E03FB6]\n\t(No symbol) [0x00DF66F0]\n\tBaseThreadInitThunk [0x7610FCC9+25]\n\tRtlGetAppContainerNamedObjectPath [0x7745809E+286]\n\tRtlGetAppContainerNamedObjectPath [0x7745806E+238]\n\n", "trace": "  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\AppData\\Local\\Programs\\Python\\Python312\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Dell\\Desktop\\Ambition hire file\\Ambition hire\\conftest.py\", line 135, in authenticated_user\n    login.get(env)\n  File \"C:\\Users\\Dell\\Desktop\\Ambition hire file\\Ambition hire\\Pages_Candidate\\base_page_candidate.py\", line 20, in get\n    self.driver.get(url)\n  File \"C:\\Users\\Dell\\AppData\\Roaming\\Python\\Python312\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 357, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\Dell\\AppData\\Roaming\\Python\\Python312\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 348, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Dell\\AppData\\Roaming\\Python\\Python312\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1739270110515, "stop": 1739270112207}], "start": 1739270110515, "stop": 1739270112660}